tests/testthat/test-financial_toy_data.R

```diff
@@ -0,0 +1,3 @@
+test_that("hasn't changed", {
+  expect_snapshot(financial_toy_data)
+})
```

Comment:
I use this approach but note it's brittle. For example, the test will break if the tibble package changes details of the print method. You can make this a little more robust with `as.data.frame()` since the print method of `data.frame` is in base R -- which is more stable.